public/shared/messages.js

```diff
@@ -31,10 +31,30 @@ const ERROR_DELETING_DATASET_MESSAGE =
 const SUCCESS_DELETING_DATASET_MESSAGE = 'The dataset was deleted successfully';
 const SUCCESS_DELETING_RESULT_MESSAGE =
   'The result dataset was deleted successfully';
-  const SUCCESS_DELETING_ALGORITHM_MESSAGE =
+const SUCCESS_DELETING_ALGORITHM_MESSAGE =
   'The algorithm was deleted successfully';
 const ERROR_DELETING_RESULT_MESSAGE =
   'There was an error deleting the result dataset';
+const ERROR_PYTHON_NOT_INSTALLED_MESSAGE =
+  'Python is not installed on your system';
```

Comment:
_Python was not found on your system. Please make sure you have Python 2.7.0 or better installed._